Sudoku/Controller/GameController.cs

```diff
@@ -12,25 +13,27 @@ public class GameController
     private readonly Game _game;
     private readonly IBoardView _boardView;
     private readonly IVisitorFactory _visitorFactory;
+    public MainController Controller { get; set; }
```

Comment:
ik wilde bijna vragen of je 'Controller'  kon hernoemen naar MainController maar daar gaat hij vgm moeilijk om doen omdat MainController al iets is... xDDD